tests/foreman/api/test_contentview.py

```diff
@@ -84,6 +82,29 @@ def content_view(module_org):
     return entities.ContentView(organization=module_org).create()
 
 
+def apply_package_filter(content_view, repo, package, inclusion=True):
+    """Apply package filter on content view
+
+    :param content_view: entity content view
+    :param repo: entity repository
+    :param str package: package name to filter
+    :param bool inclusion: True/False based on include or exclude filter
+
+    :return list : list of content view versions
+    """
+    cv_filter = entities.RPMContentViewFilter(
```

Comment:
Why this could be the part of this PR ?

If it should, then move it under api_factory and access from there that relates to the satellite object.

Reply:
Or if you want to keep it here, just pass the satellite object as a prameter here

Reply:
Since this method is being used in this module so I would like to keep it here only. I added satellite object here.